packages/components-react/src/heading/Heading.test.tsx

```diff
@@ -0,0 +1,148 @@
+import { describe, expect, it } from '@jest/globals';
+import { render, screen } from '@testing-library/react';
+import { LuxHeading, LuxHeading1, LuxHeading2, LuxHeading3, LuxHeading4, LuxHeading5, LuxHeading6 } from './Heading';
+
+describe('Dynamic Heading', () => {
```

Comment:
```suggestion
describe('Heading', () => {
```

Just keep with the component name, please